hknweb/exams/models.py

```diff
@@ -1,30 +1,39 @@
 from django.db import models
 
 class Department(models.Model):
-    name = models.CharField(unique=True, max_length=4, null=True) #short form (e.g. 'EE')
-    subject = models.CharField(max_length=255) #long form (e.g. 'Electrical Engineering')
+    abbreviated_name = models.CharField(unique=True, max_length=4, null=True) #short form (e.g. 'EE')
```

Comment:
Should we extend max_length to 8? (Just in case something weird happens and we need to support longer department abbreviations, like idk DATA or CHEM or PHYS hm wait they're all 4 letters)